### DISCUSSION TITLE
Prisma seed upsert not ordered properly

### DISCUSSION THREAD

**Discussion by ghost** [prisma/prisma#6405] [Q&A] [answered]
hi just wanted to clarify if this is a expected behaviour, here's `seed.ts` and `schema.prisma` file

```ts
import { PrismaClient } from '@prisma/client';
const prisma = new PrismaClient();
const tagsSeed = [{ name: 'diary' }, { name: 'health' }, { name: 'academic' }];

async function main() {
  for (let i = 0; i < tagsSeed.length; i++) {
    await prisma.tags.upsert({
      update: {
        name: tagsSeed[i].name,
      },
      where: {
        name: tagsSeed[i].name,
      },
      create: tagsSeed[i],
    });
  }
}

main()
  .catch((e) => {
    console.error(e);
    process.exit(1);
  })
  .finally(async () => {
    await prisma.$disconnect();
  });
```

```prisma
model Tags {
  id                Int     @default(autoincrement()) @id
  name              String  @unique
}
```
`npx prisma db seed --preview-feature` result<br>
<img width="118" alt="Screen Shot 2021-04-04 at 15 39 17" src="https://user-images.githubusercontent.com/33424694/113503416-ee72de00-955b-11eb-801d-ec3b5d1ca21f.png">
 however if i change the academic to iacademic for example, this sorted properly, is it sorted by some other column not id?
here's the example (after changing the tagsSeed academic to iacademic)
<img width="112" alt="Screen Shot 2021-04-04 at 15 42 50" src="https://user-images.githubusercontent.com/33424694/113503492-6b9e5300-955c-11eb-9d94-7cb197d8f3c8.png">

**Comment by ryands17** [comment, marked as answer]
@peedo14 👋 

I think this depends on the DB viewer that you are using. I would suggest using Prisma Studio via `npx prisma studio` and check and you will find the results ordered correctly.
